GameData/NearFutureElectrical/Patches/NFElectricalUSI.cfg

```diff
@@ -994,9 +994,63 @@
 @PART[nuclear-recycler-25]:NEEDS[MKS]
 {
 	// MKS has a different way of producing EnrichedUranium, from Uraninite,
-	// so producing it from Ore isn't appropriate.
+	// so producing it from Ore isn't appropriate. We produce it with Uranite instead.
 	!MODULE[ModuleResourceConverter]:HAS[@INPUT_RESOURCE:HAS[#ResourceName[Ore]]&@OUTPUT_RESOURCE:HAS[#ResourceName[EnrichedUranium]]]{}
 	@description,0 ^= :both process Ore into Enriched Uranium and ::
```

Comment:
Might want to change this line to something like:

    @description,0 ^= :process Ore:process Uraninite:

That way, instead of completely removing the part of the sentence that talks about Ore, it'll change it to refer to the correct resource instead.